content/md/en/docs/reference/frame-pallets.md

```diff
@@ -63,7 +60,7 @@ You should check the [Rust documentation](https://paritytech.github.io/substrate
 | [`pallet_nicks`](https://paritytech.github.io/substrate/master/pallet_nicks/index.html) | Demonstrates simplified account naming on-chain. It makes no effort to create a name hierarchy, be a DNS replacement, or provide reverse lookups.
 | [`pallet_offences`](https://paritytech.github.io/substrate/master/pallet_offences/index.html) | Tracks reported offences.
 | [`pallet_proxy`](https://paritytech.github.io/substrate/master/pallet_proxy/index.html)| Allows accounts to give permission to other accounts to dispatch types of calls from their signed origin.
-| [`pallet_randomness_collective_flip`](https://paritytech.github.io/substrate/master/pallet_randomness_collective_flip/index.html) | Provides a `random` function that can be used in tests and generates low-influence random values based on the block hashes from the previous `81` blocks. This pallet is not intended for use in production.
+| [`pallet_randomness_collective_flip`](https://paritytech.github.io/substrate/master/pallet_insecure_randomness_collective_flip/index.html) | Provides a `random` function that can be used in tests and generates low-influence random values based on the block hashes from the previous `81` blocks. This pallet is not intended for use in production.
```

Comment:
Iirc this pallet is deprecated. Some other issue was open on the docs to remove this.... I think.

Reply:
Maybe - not in the scope of this PR though;

Reply:
It is not deprecated, it is merely marked as unsafe.